app/uk/gov/hmrc/partnershipidentificationfrontend/service/ValidationOrchestrationService.scala

```diff
@@ -27,14 +27,18 @@ class ValidationOrchestrationService @Inject()(partnershipIdentificationService:
                                                validatePartnershipInformationService: ValidatePartnershipInformationService
                                               )(implicit ec: ExecutionContext) {
 
-  def orchestrate(journeyId: String)(implicit hc: HeaderCarrier): Future[ValidationResponse] =
+  def orchestrate(journeyId: String, businessVerificationCheck: Boolean)(implicit hc: HeaderCarrier): Future[ValidationResponse] =
     partnershipIdentificationService.retrievePartnershipInformation(journeyId).flatMap {
       case Some(PartnershipInformation(Some(SaInformation(sautr, postcode)), Some(_))) =>
         validatePartnershipInformationService.validateIdentifiers(sautr, postcode).flatMap {
```

Comment:
On the Incorporated Partnership journeys we currently have no way of checking that the company number they enter matches the other data. We can still do the current identifersMatch check with the postcode and sautr but because we can't confirm the company number data no matter the result of the check we store the overall value as false. Like you say it's for the future I think